hack/lib/version.sh

```diff
@@ -130,7 +130,9 @@ kube::version::ldflag() {
 kube::version::ldflags() {
   kube::version::get_version_vars
 
-  local -a ldflags=($(kube::version::ldflag "buildDate" "$(date -u +'%Y-%m-%dT%H:%M:%SZ')"))
+  local buildDate=
+  [ -n "$SOURCE_DATE_EPOCH" ] && buildDate="--date=@$SOURCE_DATE_EPOCH"
```

Comment:
is `SOURCE_DATE_EPOCH` a standard variable for these sorts of things?
otherwise, most of the kubernetes-specific variables are usually prefixed with `KUBE_`.

other minor nits: please follow the style in the rest of this file, i.e. prefer `[[ foo ]]` over `[ foo ]` and `${var}` over `$var`.